itest/rust/src/builtin_tests/script/script_instance_tests.rs

```diff
@@ -222,3 +244,112 @@ impl ScriptInstance for TestScriptInstance {
         None
     }
 }
+
+#[derive(GodotClass)]
+#[class(base = ScriptLanguageExtension, tool, init)]
+struct TestScriptLanguage {
+    base: Base<ScriptLanguageExtension>,
+}
+
+#[godot_api]
+impl TestScriptLanguage {
+    // In a real implementation the script would be created by the engine via ScriptLanguageExtension::create_script(), via a ResourceFormatLoader or by creating a new instance of TestScript. Most likely the script language would be registered as an engine singleton to make it globally available.
```

Comment:
Maybe split to multiple lines ([120-145 chars per line)](https://godot-rust.github.io/book/contribute/conventions.html#formatting).